draft-tigress-general-requirements.md

```diff
@@ -43,59 +43,38 @@ author:
+This document describes the use cases necessitating the secure transfer of digital credentials, residing in a digital wallet, between two devices and defines general assumptions, requirements and the scope for possible solutions to this problem.
 
 --- middle
 
 # Introduction
 
+In this document we are identifying a problem of transferring digital credentials (e.g. a digital car key, a digital key a a hotel room or a digital key to a private house) from a wallet on one device (smartphone) to another, particularly, if these devices belong to two different platforms (e.g. one is iOS, another - Android).
 Today, there is no widely accepted way of transferring digital credentials securely between two digital wallets independent of hardware and software manufacturer. This document describes the problem space and the requirements for the solution the working group creates.
 
-Tigress allows for a sender and receiver to communicate in order to facilitate a secure credential transfer between two digital wallets. Tigress also specifies certain privacy requirements in order to maintain a high level of user privacy.
+A Working Group, called Tigress has been established to find a solution to the problem described above.
+Within the WG an initial solution was presented (https://datatracker.ietf.org/doc/draft-art-tigress). The community decided to generalize the requirements to the solution and consider alternative solutions within the WG.
+
+This document presents the general requirements to possible solutions and  specifies certain privacy requirements in order to maintain a high level of user privacy.
 
 # General Setting
 
-When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two digital wallets, potentially through an intermediary server, there are a couple more actors involved.
+When sharing digital secure credentials, there are several actors involved. This document will focus on sharing information between two digital wallets, directly or through an intermediary server.
 
-The companies that are providing the digital credential for consumption by a digital wallet are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each digital wallet has a preexisting trust relationship between itself and the Provisioning Partner.
+Digital credentials provide access to property owned and operated by 3-rd party companies, such as hotel or residential building owners. The companies that are providing the digital credential for consumption by a digital wallet are referred to as Provisioning Partners.
+Provisioning Partner require to have control over digital credential issuance and life time management. Each digital wallet has a preexisting trust relationship between itself and the Provisioning Partner.
```

Comment:
Normally it would be authenticated access from device to Partner - based on auth by client TLS cert or some form of a token. Partner endpoints are protected from unauthorized access. I struggling with how to concisely formulate this

Reply:
Added a paragraph with devices roles